modules/store/spec/types/utils.ts

```diff
@@ -1,6 +1,6 @@
 export const compilerOptions = () => ({
   moduleResolution: 'node',
-  target: 'es2017',
+  target: 'es5',
```

Comment:
I had to change this to reproduce the issue.
To be honest, I'm not sure why because the snippet does also fail with `es2017` in the TS playground.

Reply:
I added an extra test for safety.